spec/factories/contact.rb

```diff
@@ -4,7 +4,5 @@
   factory :contact do
     email { Faker::Internet.email }
     is_subscribed { Faker::Boolean }
-
-    association :seller
```

Comment:
This association in the factory was automatically creating sellers when I created an Item via the factory. Since this factory association isn't really being used in any tests —  I'm going to remove it